code/game/objects/effects/spiders.dm

```diff
@@ -174,19 +177,22 @@
 	if(isturf(loc))
 		amount_grown += rand(0,2)
 		if(amount_grown >= 100)
-			if(SSmobs.giant_spiders > SPIDER_HARD_CAP)
+			if(SSmobs.xenobiology_mobs > MAX_GOLD_CORE_MOBS && xenobiology_spawned)
 				qdel(src)
 				return
 			if(!grow_as)
 				grow_as = pick(typesof(/mob/living/simple_animal/hostile/poison/giant_spider))
 			var/mob/living/simple_animal/hostile/poison/giant_spider/S = new grow_as(loc)
-			SSmobs.giant_spiders++
+			SSmobs.xenobiology_mobs++
```

Comment:
```suggestion
```

You're incrementing it already a few lines below, accidental?

Reply:
yeah, good catch